azure-proto-compute/azure-proto-compute.csproj

```diff
@@ -6,6 +6,10 @@
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>C:\Git\azure-proto-sdk\azure-proto-compute\obj\azure-proto-compute.xml</DocumentationFile>
```

Comment:
nit: Remove.

Reply:
this is for the workshop to produce the xml docs